apps/pie-microsite/src/_data/tokenTypes.js

```diff
@@ -7,5 +7,6 @@ module.exports = {
     COLOR: 'color',
     FONT: 'font',
```

Comment:
```suggestion
    COLOR: 'color',
     ELEVATION: 'elevation'
    FONT: 'font',
```

![image](https://user-images.githubusercontent.com/28605803/217304147-9607282d-1f31-4b91-9850-77625941f71e.png)